validator/client/wait_for_activation_test.go

```diff
@@ -345,7 +345,7 @@ func TestWaitForActivation_AccountsChanged(t *testing.T) {
 			ListenForChanges: true,
 		})
 		require.NoError(t, err)
-		err = km.RecoverAccountsFromMnemonic(ctx, constant.TestMnemonic, "", "", 1)
+		err = km.RecoverAccountsFromMnemonic(ctx, constant.TestMnemonic, "english", "", 1)
```

Comment:
It's too easy for someone to make a typo in hardcoding this "english" string in the future and we may not notice. Instead, we should define a global in some package, perhaps the accounts package, called DefaultMnemonicLanguage and set that to "english". Then, you can do `accounts.DefaultMnemonicLanguage` everywhere that hardcodes "english"